chrome/content/test/tests.js

```diff
@@ -3,43 +3,7 @@
  * It uses the Jasmine JS testing library.
```

Comment:
Rename this to match the name of the script being tested, or label it as an integration test. This may require adding additional test spec files.